plugins/module_utils/network/iosxr/config/prefix_lists/prefix_lists.py

```diff
@@ -0,0 +1,127 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+"""
+The iosxr_prefix_lists config file.
+It is in this file where the current configuration (as dict)
+is compared to the provided configuration (as dict) and the command set
+necessary to bring the current configuration to its desired end-state is
+created.
+"""
+
+from copy import deepcopy
+
+from ansible.module_utils.six import iteritems
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import (
+    dict_merge,
+)
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.resource_module import (
+    ResourceModule,
+)
+from ansible_collections.cisco.iosxr.plugins.module_utils.network.iosxr.facts.facts import (
+    Facts,
+)
+from ansible_collections.cisco.iosxr.plugins.module_utils.network.iosxr.rm_templates.prefix_lists import (
+    Prefix_listsTemplate,
+)
+
+
+class Prefix_lists(ResourceModule):
+    """
+    The iosxr_prefix_lists config class
+    """
+
+    def __init__(self, module):
+        super(Prefix_lists, self).__init__(
+            empty_fact_val={},
+            facts_module=Facts(module),
+            module=module,
+            resource="prefix_lists",
+            tmplt=Prefix_listsTemplate(),
+        )
+        self.parsers = [
+            "prefix",
+            "description",
+            "prefix_list"
+        ]
+
+    def execute_module(self):
+        """ Execute the module
+
+        :rtype: A dictionary
+        :returns: The result from module execution
+        """
+        if self.state not in ["parsed", "gathered"]:
+            self.generate_commands()
+            self.run_commands()
+        return self.result
+
+    def generate_commands(self):
+        """ Generate configuration commands to send based on
+            want, have and desired state.
+        """
+
+        wantd = self._list_to_dict(self.want)
+        haved = self._list_to_dict(self.have)
+
+
+        # if state is merged, merge want onto have and then compare
+        if self.state == "merged":
+            wantd = dict_merge(haved, wantd)
```

Comment:
Can you add the warning msg as per other platforms for merged
ref: [warning msg for merged](https://github.com/ansible-collections/cisco.nxos/pull/307/files#diff-3e585e63f353a34c61831886f8e47e3d9307a4b450ca47776a36d41c31039979R124-R126)

Reply:
This is not required in case of iosxr. as we don't need to negate a prefix-list sequence in order to update it